vumi/message.py

```diff
@@ -243,6 +257,8 @@ def validate_fields(self):
         if self['session_event'] not in self.SESSION_EVENTS:
             raise InvalidMessageField("Invalid session_event %r"
                                       % (self['session_event'],))
+        self.assert_field_value('from_addr_type', *self.ADDRESS_TYPES)
+        self.assert_field_value('to_addr_type', *self.ADDRESS_TYPES)
```

Comment:
I'm wondering whether we want to force these to be in a specific set? It seems more like something a transport should check (i.e. is this address type one that I support).